torch/csrc/distributed/c10d/ProcessGroupNCCL.hpp

```diff
@@ -89,6 +89,11 @@ static std::vector<std::string> TORCH_NCCL_TRACE_BUFFER_SIZE = {
 static std::vector<std::string> TORCH_NCCL_WAIT_TIMEOUT_DUMP_MILSEC = {
     "TORCH_NCCL_WAIT_TIMEOUT_DUMP_MILSEC"};
 
+// Control the interval inside the watchdog thread to check the signal
+// from other timeout ranks to dump the debugging information.
+static std::vector<std::string> TORCH_NCCL_TIMEOUT_CHECK_MILSEC = {
```

Comment:
this name is a bit confusing (We have several other timeout variables/envs).

TORCH_NCCL_COORDINATED_DUMP_INTERVAL_MS
something like that?

Reply:
Maybe `TORCH_NCCL_COORD_DUMP_CHECK_MS`?